src/routes/Manual.svelte

```diff
@@ -36,68 +31,41 @@ $: activeRfPower = 0;
 $: configuring = false;
 $: antennaButtonDisabled = configuring || rfPower == activeRfPower;
 $: antennaButtonMessage = "Configure";
+$: rf_command = `{ "SetRadioFreqPower": { "power_level": ${rfPower}} }`
+const reset_command = '{ "SystemReset": { } }';
 
 $: configureAttempt = {};
-
+//Need system reset after sending this command?
```

Comment:
Yes, the Feig Reader will not enforce the new RF Power setting until the next boot trigger by a system reset